### ISSUE TITLE
Embeddable URL for Dashboard widget or saved Discovery query

### ISSUE THREAD

**Issue by michaelchai-sentry** [getsentry/sentry#59157]
### Problem Statement

A user of ours would like to embed Sentry widgets and saved queries into other pages/tools they use. One example is having a Notion dashboard that tracks high-level health of their systems from multiple sources, and they would love to add Sentry visualizations there. Currently we don't have embeddable URLs for use cases like these.

### Product Area

Dashboards

┆Issue is synchronized with this [Jira Improvement](https://getsentry.atlassian.net/browse/FEEDBACK-1972) by [Unito](https://www.unito.io)

**Comment by narsaynorath** [comment]
Thanks for filing this @michaelchai-sentry, really cool idea for dashboards. For now we're going to backlog this until we get a chance to revisit it in the future

**Comment by mrloop** [comment]
Same here, we have a project dashboard, showing data from multiple sources. And it is used daily, the sentry dashboard tends to get forgotten as it can't be embedded in the project dashboard.

**Comment by zoesyc** [comment]
Thanks for the feedback @mrloop! 

We're working toward improving our dashboarding experience at Sentry and I was wondering how you use your project dashboard today. You mentioned daily monitoring, do you use the dashboard as an entry point to the multiple sources for troubleshooting? Or are you mostly monitoring for regressions?

**Comment by mrloop** [comment]
The dashboard is used as an entry point for anybody, including team members, who want to get information about the current state of the dev team for that CFT. We have multiple dashboards for different CFTs. The dashboards are similar but are managed by the team using it, so they may add or remove different data sources depending on what is important to them.
For instance the team I work on has bug tracking, current feature work kanban, run tasks, upcoming feature work and a few other things. All of these are fairly compact and you can quickly get an overview of what the team is doing and any issues, blockers, and progress on features. It's used in daily stand up with other team members to prioritise work for the day, and through out the day for easy access to that data. 
We use Sentry for error tracking and alerting. Errors automatically tagged by CFT by where they are originated and we have numerous Sentry dashboards and alerts setup in Sentry, if we go over a certain threshold for CFT an alert is raised and sent to CFT specific slack channel for further investigation. We use Sentry a lot but as a team member you need to go looking for information from Sentry, there's a link on the dashboard but we never discuss it on a daily because it's not there visually shown in the dashboard, all the other sections will get a mention, if only to says, it looks OK lets move on. Ideally I want to be able to embed the Number of Error graph from an Alert in our dashboard, the Alerts are setup per CFT and this would let us see if we've went over threshold in for the last week, and also quickly see if there are any trends, e.g. error rate increasing over last week.

**Comment by zoesyc** [comment]
Thanks for such a thorough layout of your use case!

As mentioned, we are looking to improve Dashboards within Sentry and you've touched on a couple widgets (ie. Number of Errors) and mechanisms (ie. Embeddable URLs) we're looking into. Please continue to monitor this issue - will keep it updated with any developments

**Comment by stefanoamorelli** [comment]
Embedding Sentry Dashboards in Notion pages would indeed extremely helpful and a game-changer for me and my teams!

@zoesyc

**Comment by mugen808** [comment]
+1 for embedding Sentry Dashboards in Notion pages @zoesyc. Our team is really looking up for this

**Comment by AlexJurik** [comment]
@zoesyc Our team will really appreciate prioritisation of this issue 🙏 It would really be helpful to embed Sentry to Notion!

**Comment by solidsquare** [comment]
+1 @zoesyc - This would a big help for our team as well!

**Comment by pweissbrod-eh** [comment]
+1 @zoesyc

**Comment by Dhrumil-Sentry** [comment]
Hi all!  We've added this to our backlog. We don't have a concrete timeline yet but as @zoesyc mentioned we'll comment on the issue as we have more updates.

I'd love to get the following feedback from the developers who commented very recently 🙏 

1. What type of Sentry widgets do your teams find most helpful?
2. Does embedding the dashboard mainly help your team consolidate information in 1 place? Do you use context from other tools along with Sentry's dashboard to get a better overview?

**Comment by solidsquare** [comment]
That's great to hear @Dhrumil-Sentry!

We use notion for planning and have a page where we consolidate sources of things to work on. We would like to have access to Top Issues directly in Notion so that we can see when it makes sense to take action and add it to our upcoming tasks. Thanks!

**Comment by stefanoamorelli** [comment]
Are there any updates on this one? @Dhrumil-Sentry

**Comment by ale-cota** [comment]
This feature is on our backlog and we plan to work on it in the first half of the year. It relies on other functionality that needs to be developed first, but we'll share updates as the work progresses.

**Comment by jlauzy** [comment]
Hey @ale-cota any update on this 🙏

**Comment by bcoe** [comment]
Hey @jlauzy, no major updates.

This work is still in our backlog, but it's a big feature and we have a few things ahead of it in line (_a biggie that comes to my mind, being the ability to apply filters on a dashboards without editing widgets._).

**Comment by SeriousM** [comment]
Hi, are there any updates on this? We would like to throw the dashboards on our monitor wall and don't have the ability to authenticate every time.

**Comment by bcoe** [comment]
> Hi, are there any updates on this? We would like to throw the dashboards on our monitor wall and don't have the ability to authenticate every time.

Hello @SeriousM, we are doing a little bit of work in this area, we are adding support for sentry chart unfurling in Slack.

But with regards to a publicly shareable dashboard, or the ability to embed charts in other tools (like Notion), this isn't planned in the short term.

**Comment by SeriousM** [comment]
> But with regards to a publicly shareable dashboard, or the ability to embed charts in other tools (like Notion), this isn't planned in the short term.

that's.... unfortunate... I really would like to know the business decision behind that. It's not *that* hard to enable this feature, though.

**Comment by bcoe** [comment]
@SeriousM I agree that being able to place a dashboard on a monitor and not re-authenticate would be awesome. Along with the ability to embed dashboard widgets in external applications, _which we are working on!_

All I can say, is that passionate feedback asking for functionality like this does help make a case for and prioritize work.
